service/controller/v23patch1/version_bundle.go

```diff
@@ -8,8 +8,8 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "TODO",
-				Description: "TODO",
+				Component:   "kubernetes",
+				Description: "Update kubernetes to 1.14.5 (CVE-2019-11247)",
```

Comment:
We should put a link into this description, see here for example: https://github.com/giantswarm/kvm-operator/blob/master/service/controller/v23/version_bundle.go#L12

Reply:
fixed